### PR TITLE
docs: Fix incorrect type in docs for `Parse.Query.containedIn` and `Parse.Query.notContainedIn`

### LINKED ISSUE #1783: Parse.Query.containedIn documentation issue
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/Parse-SDK-JS/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the latest versions of [Parse Server](https://github.com/parse-community/parse-server/releases) and the [Parse JS SDK](https://github.com/parse-community/Parse-SDK-JS/releases). 

### Issue Description
The Parse.Query.containedIn documentation (https://parseplatform.org/Parse-SDK-JS/api/4.0.0/Parse.Query.html#containedIn) lists the value parameter as type * but it needs to be an array otherwise error as reported in https://community.parseplatform.org/t/error-parse-error-bad-in-value/562 happens

### Steps to reproduce
Read the documentation, try a Query with
`let query = new Parse.Query(SomeObject);
query.containedIn("arrayColumn", "someValue");
`
Server will throw an error - Error 107: bad $in value

### Actual Outcome
Error 107: bad $in value

### Expected Outcome
A working query

###  Environment
Parse server 4.5.0, cloud code

Server
- Parse Server version: `4.5.0`
- Operating system: `Server`
- Local or remote host (AWS, Azure, Google Cloud, Heroku, Digital Ocean, etc): `Remote: Back4App`

Database
- System (MongoDB or Postgres): `MongoDB`
- Database version: `3.6`
- Local or remote host (MongoDB Atlas, mLab, AWS, Azure, Google Cloud, etc): `Remote: Back4App`

Client
- Parse JS SDK version: `Don't know - any way I can query it from the cloud code? `

### PR DESCRIPTION
**Pull request by richardgroves**
## Pull Request

- Report security issues [confidentially](https://github.com/parse-community/Parse-SDK-JS/security/policy).
- Any contribution is under this [license](https://github.com/parse-community/Parse-SDK-JS/blob/alpha/LICENSE).
- Link this pull request to an [issue](https://github.com/parse-community/Parse-SDK-JS/issues?q=is%3Aissue).

## Issue

Closes: #1783 

## Approach

Documentation and method parameter type updated for `Parse.Query.containedIn` and `Parse.Query.notContainedIn`

## Tasks

- [x] Add changes to documentation (guides, repository pages, code comments)

### CODE REVIEW

**Comment by richardgroves** [comment]
> ### Thanks for opening this pull request!
> * ❌ Please edit your post and use the provided template when creating a new pull request. This helps everyone to understand your post better and asks for essential information to quicker review the pull request.

I'm not sure which bit of the template I didn't use, AFAICS I filled in each section.

**Comment by mtrezza** [comment]
All good, I removed the "Fixes https://github.com/parse-community/Parse-SDK-JS/issues/1783" at the top, that's what the "Closes:" is for. And there's a bug in the bot I guess because this is a new template, let me fix that real quick...

File: src/ParseQuery.js [hunk 1]
```diff
@@ -1284,10 +1284,10 @@ class ParseQuery {
    * be contained in the provided list of values.
    *
    * @param {string} key The key to check.
-   * @param {*} value The values that will match.
+   * @param {array} value The values that will match.
```
Comment (mtrezza) [inline src/ParseQuery.js:1287]:
```suggestion
   * @param {Array<String>} value The values that will match.
```

Comment (mtrezza) [inline src/ParseQuery.js]:
```suggestion
   * @param {Array<*>} value The values that will match.
```

File: src/ParseQuery.js [hunk 2]
```diff
@@ -1296,10 +1296,10 @@ class ParseQuery {
    * not be contained in the provided list of values.
    *
    * @param {string} key The key to check.
-   * @param {*} value The values that will not match.
+   * @param {array} value The values that will not match.
```
Comment (mtrezza) [inline src/ParseQuery.js]:
```suggestion
   * @param {Array<*>} value The values that will not match.
```

**Review by mtrezza — CHANGES_REQUESTED** [review]
The tests fail because in JSDos it's `Array`. It may be more correct to specify the exact types that can be passed, unless really *any* type can be passed, including objects, array, etc.

`Array<null|undefined|String|Number|Boolean>` is just a guess

**Comment by richardgroves** [comment]
> The tests fail because in JSDos it's `Array`. It may be more correct to specify the exact types that can be passed, unless really _any_ type can be passed, including objects, array, etc.
> 
> `Array<null|undefined|String|Number|Boolean>` is just a guess

I'm not 100% sure what types can be in the array, and whether an array of items has to be of a single type or can have mixed contents. Would specifying the type list limit it to an array of homogenous types?

I know I have arrays of objects (or object pointers internally) so 'object' would have to be in the list of allowed types.

**Comment by mtrezza** [comment]
> Would specifying the type list limit it to an array of homogenous types? I know I have arrays of objects (or object pointers internally) so 'object' would have to be in the list of allowed types.

Yes. So let's go with `Array<*>` I guess. Let's see whether the CI passes this time...

**Review by mtrezza — APPROVED** [review]
Looks good!

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [4.1.0-beta.1](https://github.com/parse-community/Parse-SDK-JS/releases/tag/4.1.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [4.1.0-alpha.1](https://github.com/parse-community/Parse-SDK-JS/releases/tag/4.1.0-alpha.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [4.1.0](https://github.com/parse-community/Parse-SDK-JS/releases/tag/4.1.0)

**Pull request merged**